front/src/applications/stdcm/types.ts

```diff
@@ -22,16 +21,30 @@ export type StdcmV2SuccessResponse = Omit<
 > & {
   simulation: Extract<SimulationResponse, { status: 'success' }>;
   rollingStock: LightRollingStock;
+  creationDate: Date;
+  speedLimitByTag?: string;
 };
 
 export type SimulationReportSheetProps = {
   stdcmData: StdcmV2SuccessResponse;
-  pathProperties?: ManageTrainSchedulePathProperties;
-  rollingStockData: RollingStockWithLiveries;
-  speedLimitByTag?: string;
   simulationReportSheetNumber: string;
   mapCanvas?: string;
-  creationDate?: Date;
+  operationalPointsList: stdcmResultsOperationalPointsList;
+};
+
+export type stdcmResultsOperationalPointsList = stdcmResultsOperationalPoint[];
+
+type stdcmResultsOperationalPoint = {
```

Comment:
Types usually start by a capital letter
```suggestion
export type StdcmResultsOperationalPointsList = StdcmResultsOperationalPoint[];

type StdcmResultsOperationalPoint = {
```

Reply:
fix